web/src/test/manual/web/issue2924/source/dmg.dv.test/dmg.dv.test.kpj

```diff
@@ -48,13 +48,6 @@
       <FileVersion></FileVersion>
       <FileType>.md</FileType>
     </File>
-    <File>
-      <ID>id_4b12eec2a4d0bfc7b5441a4a59691eab</ID>
-      <Filename>dmg.dv.test.model_info</Filename>
-      <Filepath>dmg.dv.test.model_info</Filepath>
-      <FileVersion></FileVersion>
-      <FileType>.model_info</FileType>
-    </File>
```

Comment:
Should we also clean up .keyboard_info files in other web test folders?

web/srce/test/manual/web/issue917-context-and-notany/test_917/test_917.keyboard_info
web/srce/test/manual/web/issue2924/source/testvariable/testvariable.keyboard_info
web/srce/test/manual/web/issue3701/test3701/test3701.keyboard_info

Reply:
See #9551